tests/unit/Storage/ABS/SynapseExportAdapterTest.php

```diff
@@ -104,11 +104,12 @@ public function testGetCopyCommandQuery(): void
 
         $source = new Storage\Synapse\SelectSource('SELECT * FROM "schema"."table" WHERE id = ?', [1]);
         $adapter = new Storage\ABS\SynapseExportAdapter($conn);
-        $adapter->runCopyCommand(
+
+        $this->assertIsArray($adapter->runCopyCommand(
```

Comment:
Tohle jsi nic neoassertoval. Tohle ti hlídají typehinty a phpstan. Měl bys assertovat, že je to vždycky []